dbms/src/Storages/DeltaMerge/tests/gtest_segment_ingest.cpp

```diff
@@ -67,10 +67,19 @@ try
     ASSERT_EQ(20, getSegmentRowNum(DELTA_MERGE_FIRST_SEGMENT_ID));
     ASSERT_EQ(22, getSegmentRowNum(*right_seg));
 
-    ASSERT_TRUE(storage_pool->log_storage_v3 != nullptr);
-    storage_pool->data_storage_v3->gc(/* not_skip */ true);
-    auto stable_page_ids = storage_pool->data_storage_v3->getAliveExternalPageIds(NAMESPACE_ID);
-    ASSERT_EQ(1, stable_page_ids.size());
+    ASSERT_TRUE(storage_pool->log_storage_v3 != nullptr || storage_pool->uni_ps != nullptr);
+    if (storage_pool->log_storage_v3)
```

Comment:
should we replace these calls by `getPageNumAfterGC`?

Reply:
Fixed.